src/browser/context-menu-template.js

```diff
@@ -0,0 +1,99 @@
+var reload = function(item, focusedWindow) {
+  if (focusedWindow) {
+    focusedWindow.webContents.reload();
+  }
+};
+
+var windows_linux_template = [
```

Comment:
Man, this seems like a lot of duplication just to show keyboard shortcuts. :frowning: